web/src/state_data.ts

```diff
@@ -105,7 +105,7 @@ export const realm_schema = z.object({
     max_avatar_file_size_mib: z.number(),
     max_icon_file_size_mib: z.number(),
     max_logo_file_size_mib: z.number(),
-    max_topic_length: z.number(),
```

Comment:
Why have you deleted this?

Reply:
I didn't delete it. The change caused a conflict on my PR which I resolved as above. That change was made in another PR by another contributor. You can try checking recently merged commit for this.